spec/support/environment_spec_helper.rb

```diff
@@ -1,6 +1,8 @@
 module EnvironmentSpecHelper
   # @todo rewrite using with_env_vars
   def with_18f_procurement_env_variables
+    ENV['DISABLE_SANDBOX_WARNING'] = 'true'
```

Comment:
I know you were just following what the rest of the rest of the method is doing, but mind converting all of them to use `with_env_vars`?